backend/index.js

```diff
@@ -1,54 +1,105 @@
-const express = require('express');
-const cors = require('cors');
-const helmet = require('helmet');
-const http = require('http');
-const responseTime = require('response-time');
-const routes = require('./app/routes');
-const { errorHandler } = require('./helpers/error');
-require('colors');
-require('./helpers/dbConnection');
 require('dotenv').config();
+const cluster = require('cluster');
+const os = require('os');
+
+if (process.env.CLUSTER === 'yes') {
+  const numCPUs = os.cpus().length;
+
```

Comment:
refactor this to a new file, just for the cluster code

Reply:
I have added the the code to run in both cluster and non cluster mode through the env.

So if someone wants to run in non cluster mode he can just change his env file. This will ensure common entry point for the server.